snapshot/lib/snapshot/test_command_generator.rb

```diff
@@ -64,7 +64,7 @@ def pipe
         ["| tee #{xcodebuild_log_path.shellescape} | xcpretty #{Snapshot.config[:xcpretty_args]}"]
       end
 
-      def device_udid(device)
```

Comment:
This method is called from `snapshot/runner.rb`, right? Or did I miss something?